ui/src/app/streams/stream-deploy/stream-deploy.component.ts

```diff
@@ -181,15 +188,17 @@ export class StreamDeployComponent implements OnInit, OnDestroy {
     this.update(value);
     const propertiesMap = {};
     value.forEach((val) => {
-      const arr = val.split(/=(.*)/);
-      if (arr.length !== 3) {
-        console.error('Split line property', val);
-      } else {
-        // Workaround sensitive property: ignored property
-        if (arr[1] === `'******'`) {
-          console.log(`Sensitive property ${arr[0]} is ignored`);
+      if (this.originalProperties.indexOf(val) === -1) {
```

Comment:
This seems to be the reason why it is not working, yet. While debugging this I saw this:

![image](https://user-images.githubusercontent.com/685363/40397649-71d4845a-5dcf-11e8-9402-e4efc5345422.png)